app/src/main/java/io/horizontalsystems/bankwallet/entities/LogEntry.kt

```diff
@@ -0,0 +1,15 @@
+package io.horizontalsystems.bankwallet.entities
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity
+data class LogEntry(
+        var date: Long,
+        var level: Int,
+        var actionId: String,
+        var message: String
+) {
+    @PrimaryKey(autoGenerate = true)
+    var id: Int? = null
```

Comment:
Can we go without optional here?
`var id: Int = 0`

Reply:
I think it's better to use null for this case.